internal/libs/crypto/options.go

```diff
@@ -0,0 +1,57 @@
+package crypto
+
+// getOpts - iterate the inbound Options and return a struct.
+func getOpts(opt ...Option) options {
```

Comment:
As this is a new package, you may want `getOpts` to return `(options, error)`

Reply:
done.